base/slicearray.jl

```diff
@@ -85,9 +85,10 @@ the ordering of the dimensions will match those in `dims`. If `drop = false`, th
 `Slices` will have the same dimensionality as the underlying array, with inner
 dimensions having size 1.
 
-See [`stack`](@ref)`(slices; dims)` for the inverse of `eachcol(A; dims::Integer, drop=true)`.
+For matrices, the case `dims = 1` is [`eachrow`](@ref), and `dims = 2` is [`eachcol`](@ref).
 
-See also [`eachrow`](@ref), [`eachcol`](@ref), [`mapslices`](@ref) and [`selectdim`](@ref).
+[`stack`](@ref)`(slices; dims)` is the inverse of `eachslice(A; dims::Integer, drop=true)`.
+See also [`mapslices`](@ref) and [`selectdim`](@ref).
 
```

Comment:
Here "the inverse of `eachcol(A; dims::Integer, drop=true)`" was a typo in https://github.com/JuliaLang/julia/pull/46429 -- eachcol does not take keywords